src/header.c

```diff
@@ -0,0 +1,141 @@
+#include "header.h"
+#include "defines.h"
+#include "footer.h"
+
+#include <string.h>
+#include <endian.h>
+#include <stdio.h>
+
+
+/*!
+ * \brief buildISOHeader Constructs an ISO header based on the supplied message ID and content length
+ * \param id Message ID of the message for which the header is to be used
+ * \param messageLength Length of the message including header and footer
+ * \param debug Flag for enabling debugging
+ * \return A struct containing ISO header data
+ */
+HeaderType buildISOHeader(enum ISOMessageID id, uint32_t messageLength, const char debug) {
+	HeaderType header;
+
+	header.syncWord = ISO_SYNC_WORD;
+	header.transmitterID = getTransmitterID();
+	header.receiverID = 0; // TODO: Set receiver ID
```

Comment:
Set receiverID is only needed if we're going to support message gating. 
Shall we support this mechanism in ATOS? - a topic for discussion...